test/faktory_worker/worker/server_integration_test.exs

```diff
@@ -1,12 +1,21 @@
 defmodule FaktoryWorker.Worker.ServerIntegrationTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
 
   alias FaktoryWorker.Worker.Server
   alias FaktoryWorker.Random
+  alias FaktoryWorker.TestQueueWorker
+
+  # todo: flush before each test
```

Comment:
Does this need to be done?

Reply:
In the end technically not since I ended up being able to remove the test that was conflicting, however it would be more ideal so I think I will add that.